src/blocks/social-profiles/index.php

```diff
@@ -0,0 +1,257 @@
+				esc_attr($has_padding)
+			);
+		}
+	}
+
+	// Build classes.
+	$class = 'wp-block-coblocks-social wp-block-coblocks-social-profiles';
+
+	if (isset($attributes['className'])) {
+		$class .= ' ' . $attributes['className'];
+	}
+
+	if (isset($attributes['hasColors']) && $attributes['hasColors']) {
+		$class .= ' has-colors';
+	}
+
+	if (isset($attributes['size']) && 'med' !== $attributes['size'] && (isset($attributes['className']) && strpos($attributes['className'], 'is-style-mask') === false)) {
+		$class .= ' has-button-size-' . $attributes['size'];
+	}
+
+	// Render block content.
+	$block_content = sprintf(
+		'<div class="%1$s" %2$s><ul>%3$s</ul></div>',
+		esc_attr($class),
+		esc_attr($text_align),
+		$markup
+	);
+
+	return $block_content;
+}
+
+/**
+ * Registers the block on server.
+ */
+function coblocks_register_social_profiles_block()
+{
+
+	// Return early if this function does not exist.
+	if (!function_exists('register_block_type')) {
+		return;
+	}
+
+	register_block_type(
+		'coblocks/social-profiles',
+		array(
+			'editor_script'   => 'coblocks-editor',
+			'editor_style'    => 'coblocks-editor',
+			'style'           => 'coblocks-frontend',
+			'attributes'      => array(
+				'className'             => array(
+					'type'    => 'string',
+				),
+				'hasColors'             => array(
+					'type'    => 'boolean',
+					'default' => true,
+				),
+				'borderRadius'          => array(
+					'type'    => 'number',
+					'default' => 40,
+				),
+				'size'                  => array(
+					'type'    => 'string',
+					'default' => 'med',
+				),
+				'iconSize'              => array(
+					'type'    => 'number',
+					'default' => 22,
+				),
+				'padding'               => array(
+					'type'    => 'number',
+					'default' => 16,
+				),
+				'textAlign'             => array(
+					'type'    => 'string',
+				),
+				'backgroundColor'       => array(
+					'type'    => 'string',
+				),
+				'customBackgroundColor' => array(
+					'type'    => 'string',
+				),
+				'textColor'             => array(
+					'type'    => 'string',
+				),
+				'customTextColor'       => array(
+					'type'    => 'string',
+				),
+				'facebook'              => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'twitter'               => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'instagram'              => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'pinterest'             => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'linkedin'              => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'youtube'                => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'yelp'                => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'houzz'                 => array(
+					'type'    => 'string',
+					'default' => '',
+				),
+				'shareAttributes'		=> array(
```

Comment:
Same comment with the socialProfileAttributes on the share block.

Reply:
Rewrote transforms and removed cross-block attributes in favor of localStorage use.